<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>863</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1020</width>
    <height>863</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1020</width>
    <height>863</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Visão Computacional - Trab 01</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="menu_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1020</width>
      <height>353</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Menu_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>1020</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Dyuthi</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string>MENU</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QFrame" name="transformarFrame">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>50</y>
       <width>340</width>
       <height>303</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="R_label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>ROTACIONAR</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="getRX">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>60</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="RX_label">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>40</y>
        <width>30</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="RY_label">
      <property name="geometry">
       <rect>
        <x>155</x>
        <y>40</y>
        <width>30</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="getRY">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>60</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="RZ_label">
      <property name="geometry">
       <rect>
        <x>245</x>
        <y>40</y>
        <width>30</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="getRZ">
      <property name="geometry">
       <rect>
        <x>235</x>
        <y>60</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="getTX">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>160</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="T_label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>160</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>TRANSLADAR [º]</string>
      </property>
     </widget>
     <widget class="QLabel" name="TZ_label">
      <property name="geometry">
       <rect>
        <x>245</x>
        <y>140</y>
        <width>30</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="getTZ">
      <property name="geometry">
       <rect>
        <x>235</x>
        <y>160</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="getTY">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>160</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="TY_label">
      <property name="geometry">
       <rect>
        <x>155</x>
        <y>140</y>
        <width>30</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="TX_label">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>140</y>
        <width>30</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Ref_label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>REFERENCIAL</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="refWORLD">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>240</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Mundo</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="refCAM">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>240</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Câmera</string>
      </property>
     </widget>
     <widget class="QPushButton" name="transformar">
      <property name="geometry">
       <rect>
        <x>185</x>
        <y>270</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Transformar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resetTransformar">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>270</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="optionsFrame">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>340</width>
       <height>303</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="optionLabel">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>10</y>
        <width>195</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>OPÇÕES DE CÂMERA</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="getDFocal">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>80</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="dFocal_label">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>50</y>
        <width>190</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Distância Focal [mm]</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resetOptions">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QPushButton" name="alterarOptions">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>270</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Alterar</string>
      </property>
     </widget>
     <widget class="QLabel" name="tCCD_label">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>120</y>
        <width>190</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Tamanho CCD [mm]</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="getCCD_X">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="getCCD_Y">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>150</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="getIMG_Y">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>220</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="getIMG_X">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>220</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="tIMG_label">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>190</y>
        <width>230</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Tamanho Imagem [pixels]</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="_3D_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>355</y>
      <width>510</width>
      <height>510</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="_2D_frame">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>355</y>
      <width>510</width>
      <height>510</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>getDFocal</tabstop>
  <tabstop>getCCD_X</tabstop>
  <tabstop>getCCD_Y</tabstop>
  <tabstop>getIMG_X</tabstop>
  <tabstop>getIMG_Y</tabstop>
  <tabstop>getRX</tabstop>
  <tabstop>getRY</tabstop>
  <tabstop>getRZ</tabstop>
  <tabstop>getTX</tabstop>
  <tabstop>getTY</tabstop>
  <tabstop>getTZ</tabstop>
  <tabstop>refWORLD</tabstop>
  <tabstop>refCAM</tabstop>
  <tabstop>resetOptions</tabstop>
  <tabstop>alterarOptions</tabstop>
  <tabstop>resetTransformar</tabstop>
  <tabstop>transformar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
